Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/DictationInputDeviceManager.cs

```diff
@@ -97,9 +60,53 @@ public override void Dispose()
         /// </summary>
         private static AudioClip dictationAudioClip;
 
-        private static readonly WaitForUpdate NextUpdate = new WaitForUpdate();
+        public DictationInputDeviceManager(string name, uint priority) : base(name, priority) { }
+
+
+#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+        /// <inheritdoc />
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            InputSource = InputSystem?.RequestNewGenericInputSource($"Dictation Recognizer", null);
+            dictationResult = string.Empty;
+
+            dictationRecognizer = new DictationRecognizer();
+            dictationRecognizer.DictationHypothesis += DictationRecognizer_DictationHypothesis;
+            dictationRecognizer.DictationResult += DictationRecognizer_DictationResult;
+            dictationRecognizer.DictationComplete += DictationRecognizer_DictationComplete;
+            dictationRecognizer.DictationError += DictationRecognizer_DictationError;
+
+            // Query the maximum frequency of the default microphone.
+            int minSamplingRate; // Not used.
+            Microphone.GetDeviceCaps(DeviceName, out minSamplingRate, out samplingRate);
+
+            Run();
```

Comment:
We can remove this and replace run with the Update method call from MR Manager

Reply:
Done in both speech and dictation